packages/test/src/integration-tests-old.ts

```diff
@@ -398,7 +398,7 @@ export function runIntegrationTests(codec?: PayloadCodec): void {
     await t.throwsAsync(workflow.query('not found'), {
       instanceOf: QueryNotRegisteredError,
       message:
-        'Workflow did not register a handler for not found. Registered queries: [__stack_trace __enhanced_stack_trace isBlocked]',
+        'Workflow did not register a handler for not found. Registered queries: [__stack_trace __enhanced_stack_trace __temporal_workflow_metadata isBlocked]',
```

Comment:
That's getting messy. I think we should consider not listing "internals" queries.

Reply:
It could be useful for knowing what built-in queries, like `__temporal_workflow_metadata`, are supported by the WF. 
Also the UI today may expect some handlers present when parsing the error message...

Reply:
UI doesn't and shouldn't _expect_ specific handlers to be present in the list returned on query not found error message. Using that error message to report existing queries is hack, not normalized across SDKs, and going forward, UI should call `__temporal_workflow_metadata` directly anyway.

Reply:
I agree with you on the UI, but unfortunately, depending on how the UI parses the query today, we may break it if we return an empty set. It is also unfortunate that `isBlocked` does not have the `__` internal prefix. Once they switch, we can do the cleanup with them...

Reply:
:+1: Fixing the query-failed string to not include all queries anymore should be a separate task once we feel comfortable enough modern UIs are no longer using it (i.e. fixed version has been out there a while).

Reply:
My point was not about query-failed no longer returning any queries, but only about not returning SDK _built-in_ queries. For example, if using an old UI, they should not see `__stack_trace`, `__enhanced_stack_trace` and `__temporal_workflow_metadata` in the query drop down. Returning user queries is still fine, for now.

Reply:
If needed that's easy to filter by the UI using the `__` prefix. I like that we can quickly know what internal queries are supported for a given workflow...